packages/design-system/src/components/Promo/style.scss

```diff
@@ -92,7 +92,7 @@ $base-class: 'promo';
     }
 
     .#{$base-class}__wrapper {
-      max-width: 520px;
+      max-width: 680px;
```

Comment:
Banner should be changed as well. The same max-width value was there and I expect the same issue with spacings

Reply:
done, jo approved